src/index.js

```diff
@@ -5,7 +5,7 @@ import {simulations, speedy, css} from 'glamor'
 import theme from './catalogTheme'
 import './global.css'
 import './catalogTheme.css'
-import {fontStyles} from './components/Typography'
+import * as fontStyles from './components/Typography/styles'
```

Comment:
Why was this necessary?

Reply:
can't set `speedy(false)` after `css` from `glamor` has been used.  `Typography/index` uses it